package.json

```diff
@@ -55,6 +56,7 @@
     "deploy:pr": "yarn build:release:rinkeby && scripts/deployTravisBuildToSurge.sh",
     "deploy": "/bin/sh -c 'if [ \"$TRAVIS_PULL_REQUEST\" = \"false\" ]; then yarn deploy:branch; else yarn deploy:pr; fi'",
     "test": "react-scripts test --env=jsdom",
+    "cypress:open": "cypress open",
```

Comment:
Is this how we run the test? Is there anyway we can use headless browser (eg: http://phantomjs.org/) so that we can run it as part of CI?

Reply:
Agreed, would like to see this run in CI.

Reply:
Cypress are UI tests, so there is value actually looking at it. I'm sure it can also run in CI as well, but we can raise that as a separate ticket

Reply:
> @jefflau Is cypress/integration/create.spec.js the only change related to this test? There are lots of files which seems just a format change which makes it very difficult to review. Can you separate out PR to only the relevant ones (or list down all the functional and cosmetic changes) so that I can follow.

I'll have a look at prettifying on commit. However I did change quite a few files just to get our local test environment working, so it's not just cosmetic changes